src/index.html

```diff
@@ -16,7 +16,44 @@
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
```

Comment:
The 'alt' attribute for the logo image is set to 'Logo'. It's recommended to use a more descriptive text to improve accessibility, such as 'Company Logo' or 'Moyo Logo'.